fullstop-plugins/fullstop-lambda-plugin/src/main/java/org/zalando/stups/fullstop/plugin/lambda/LambdaPlugin.java

```diff
@@ -31,13 +33,23 @@
     private static final String S3_KEY_CODE_JSON_PATH = "$.code.s3Key";
     private static final String S3_KEY_JSON_PATH = "$.s3Key";
     private static final String FUNCTION_NAME_JSON_PATH = "$.functionName";
+    private static final String CODE_SHA_256_JSON_PATH = "$.codeSha256";
 
     private static final String S3_BUCKET = "s3_bucket";
     private static final String S3_KEY = "s3_key";
     private static final String FUNCTION_NAME = "function_name";
 
     private static final String EMPTY = "";
 
+    private List<String> whitelistedSha256 = newArrayList(
```

Comment:
I'd either make this a Set<String> or Map<String,String> (mapping from runtime to codeSHA). But Set is fine, also. Just seems more appropriate and performant for this use case than List